deploy/app-insights/index.js

```diff
@@ -10,5 +10,52 @@ Script Parameters
 4: Release Description
 5: Release Author
 */
-const axios = require('axios')
+const axios = require('axios').default
+const yargs = require('yargs').argv
+const uuid = require('uuid').v4
+const moment = require('moment')
+
+const applicationId = yargs.applicationId
+const apiKey = yargs.apiKey
+const releaseName = yargs.releaseName
+const releaseDescription = yargs.releaseDescription
+const author = yargs.releaseAuthor
+const eventTime = moment().format('YYYY-MM-DD[T]HH:MM:SS')
```

Comment:
I think you mean `YYYY-MM-DD[T]HH:MM:ss.SSS` or (with timezone offset) `YYYY-MM-DD[T]HH:MM:ss.SSSZZ` or `.toISOString()`

Reply:
good catch, thanks